src/clr-core/alert/alert.stories.ts

```diff
@@ -65,7 +65,7 @@ export const API = () => {
       .iconTitle=${iconTitle}
       .size=${size} 
       .status=${alertStatus}
-      @closedChange=${action('closeChanged')}>
+      @closedChange=${action('closedChange')}>
```

Comment:
I'm wondering if we should deviate from Angular event name and call it "clickedClose" because we aren't actually going to change the state of the component to closed. Thoughts on this? Same question for alerts...

Reply:
This is not exclusive to Angular most frameworks encourage a 1:1 mapping between a property and event as well as events being present tense. Since this is also tied to keyboard users so click I don't think is quite right as well. I would keep the same convention and update it to be `closeChange`. This will help ensure our properties and events are consistent as well as keep `close` available as a property if we need it in the future (hopefully not). The convention will be especially important in the future with tooling support for things like React wrappers which have to parse the events.

```html
<!-- Vue -->
<cds-modal v-bind:close="closeValue" v-on:closeChange="doSomething()">
<cds-modal :close="closeValue" @closeChange="doSomething()">

<!-- React Wrapper -->
<CdsModal close={closeValue} onClose={() => this.doSomething()} />

<!-- Svelte native & html -->
<CdsModal bind:close="closeValue" on:close={doSomething} />
<cds-modal bind:close="closeValue" on:closeChange={doSomething}>

<!-- Angular -->
<cds-modal [close]="closeValue" (closeChange)="doSomething()">

<!-- Angular JS -->
<cds-modal close="$ctrl.closeValue" on-close-change="$ctrl.doSomething()">
```

Reply:
Thank you both. Good point about it also mapping to keyboard event. I'll update modal and alerts to `closeChange` (from `closedChange` in the case of alert).